india_compliance/gst_india/utils/exporter.py

```
@@ -126,6 +154,28 @@ def create(
 
         self.apply_conditional_formatting(add_totals)
 
+    def insert_data(
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)